README.md

```diff
@@ -12,90 +12,110 @@
+- **Feature flags:** test and manage the rollout of new features
+- **Heatmaps:** see where users are using your product with the [PostHog Toolbar](https://posthog.com/docs/features/toolbar)
+- **Automated analysis:** find [correlations](https://posthog.com/docs/user-guides/correlation) between successful users and their behaviors or attributes
 
-<img src="https://posthog-static-files.s3.us-east-2.amazonaws.com/Documentation-Assets/action-toolbar.gif" width="100%">
+### Data and infrastructure tools
 
-## Getting the most of PostHog
+- **Complete control over your data:** [host it yourself](https://posthog.com/docs/self-host/overview#deploy) on any infrastructure
+- **Import and export your data:** import from and export to the services that matter to you with [apps](https://posthog.com/apps)
+- **Ready-made libraries:** we’ve built libraries for [JavaScript](https://posthog.com/docs/integrations/js-integration), [Python](https://posthog.com/docs/integrations/python-integration), [Ruby](https://posthog.com/docs/integrations/ruby-integration), [Node](https://posthog.com/docs/integrations/node-integration), [Go](https://posthog.com/docs/integrations/go-integration), [Android](https://posthog.com/docs/integrations/android-integration), [iOS](https://posthog.com/docs/integrations/ios-integration), [PHP](https://posthog.com/docs/integrations/php-integration), [Flutter](https://posthog.com/docs/integrations/flutter-integration), [React Native](https://posthog.com/docs/integrations/react-native-integration), [Elixir](https://posthog.com/docs/integrations/elixir-integration), [Nim](https://github.com/Yardanico/posthog-nim), and an [API](https://posthog.com/docs/integrations/api) for anything else
+- **Plays nicely with data warehouses:** import events or user data from your warehouse by writing a simple transformation plugin, and export data with pre-built apps - such as [BigQuery](https://posthog.com/apps/bigquery-export), [Redshift](https://posthog.com/apps/redshift-export), [Snowflake](https://posthog.com/apps/snowflake-export), and [S3](https://posthog.com/apps/s3-expo)
 
-See [PostHog Docs](https://posthog.com/docs/) for in-depth walk-throughs on functionality.
+[Read a full list of PostHog features](https://posthog.com/features).
 
-Join our [Slack community](https://posthog.com/slack) if you need help, want to chat, or are thinking of a new feature. We're here to help - and to make PostHog even better.
+## Docs and support
 
-## Philosophy
+Read how to [deploy](https://posthog.com/docs/self-host), [integrate](https://posthog.com/docs/integrate), and [extend](https://posthog.com/docs/apps) PostHog in our [documentation](https://posthog.com/docs).
 
-We help you understand user behavior and build better products without losing control of your data. 
+Check out our [tutorials](https://posthog.com/docs/apps) for step-by-step guides, how-to's, and best practices.
```

Comment:
May need some art assist from Lottie, but I think worth adding the tutorial cover images in as clickable items which break up the text and point to especially good/relevant tutorials. Defer to you.